### PR TITLE
Basic verion of footer with layout added

### PR DESCRIPTION
**Pull request by bartKarb**
I just added first version of footer - some adjustmenst might be needed

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,15 +3,19 @@ import './App.scss';
...
+    <Router>
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route index element={<Home />} />
+          <Route path="phones" element={<PhonesPage />} />
+          {/* <Route path="*" element={<ErrorPage />} /> */}
+        </Route>
+      </Routes>
+    </Router>
+    <Footer />
```
Comment (TideTwo) [inline src/App.tsx]: the footer should be in the layout component

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Link } from 'react-router-dom';
+import './Footer.scss';
```
Comment (TideTwo) [inline src/components/Footer/Footer.tsx]: you are not using scss modules

**Review by TideTwo — CHANGES_REQUESTED** [review]

File: src/components/Layout/Layout.tsx [hunk 3]
```diff
@@ -11,7 +12,7 @@ export default function Layout({ }: Props) {
         <Outlet />
       </ProductsProvider>
 
-      {/* <Footer /> */}
+      {<Footer />}
```
Comment (TideTwo) [inline src/components/Layout/Layout.tsx]: why are you using braces

File: public/icons/logo.png
Comment (TideTwo) [inline public/icons/logo.png:1]: still add images .icons when there is alr a file and icon component

**Review by TideTwo — CHANGES_REQUESTED** [review]
2 things to correct, everything else looks good

File: public/icons/logo.png
Comment (bartKarb) [reply public/icons/logo.png:1]: I just corrected mentioned things

Comment (bartKarb) [reply public/icons/logo.png:1]: I set up formatting for mobile for now. It works well, however refactoring into desktop in future is needed.

**Pull request closed without merging**